platforms/core-runtime/build-state/src/main/java/org/gradle/internal/buildprocess/BuildProcessState.java

```diff
@@ -42,7 +42,8 @@ public BuildProcessState(
         ServiceRegistryBuilder builder = ServiceRegistryBuilder.builder()
             .scopeStrictly(Scope.Global.class)
             .displayName("Global services")
-            .provider(new GlobalScopeServices(longLiving, agentStatus, additionalModuleClassPath));
+            .provider(new GlobalScopeServices(longLiving, agentStatus, additionalModuleClassPath))
```

Comment:
GlobalScopeServices should be busted up, but this is a bit complicated, so in this PR leave it mostly unchanged.